src/resolutionmodel.h

```diff
@@ -25,10 +26,18 @@ class ResolutionModel : public QAbstractListModel
 
     Q_INVOKABLE QSize sizeToRatio(const QSize &siz) const;
     Q_INVOKABLE void setImageCapture(QObject *camera);
+    Q_INVOKABLE void setVideoRecorder(QObject *capture);
+    Q_INVOKABLE void setMode(const QString &mode);
+    Q_INVOKABLE QSize defaultResolution(const QString &mode);
+    Q_INVOKABLE bool isValidResolution(const QSize &resolution, const QString &mode);
 
 private:
     QMap<QString, QSize> m_resolutions;
+    QList<QSize> m_supportedImageResolutions;
+    QList<QSize> m_supportedVideoResolutions;
     QCameraImageCapture *m_capture = nullptr;
+    QMediaRecorder *m_recorder = nullptr;
```

Comment:
These are not really needed to be stored because these are only used when changing camera.